docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -34,16 +45,16 @@ The following example shows how to create a gallery-like flexbox layout using th
     Window.Instance.Add(flexContainer);
     ```
 
-2.  To make the toolbar and main content display vertically, set the flex direction of the main container to **column**:
+3.  To make the toolbar and main content display vertically, set the flex direction of the main container to **column**:
 
-    ```
+    ```csharp
     /// Display toolbar and content vertically
     flexContainer.FlexDirection = FlexContainer.FlexDirectionType.Column;
     ```
 
-3.  Create a new flex container as the toolbar and add it to the main container. Because the flex direction in the main container is **column**, the toolbar is placed at the top.
+4.  Create a new flex container as the toolbar and add it to the main container. Since the flex direction in the main container is set to **column**, the toolbar is placed at the top.
```

Comment:
Kindly add : instead of . if the sentence is immediately followed by code. 
```suggestion
4.  Create a new flex container as the toolbar and add it to the main container. Since the flex direction in the main container is set to **column**, the toolbar is placed at the top: